src/Elastic.Apm/Helpers/StringExtensions.cs

```diff
@@ -0,0 +1,18 @@
+namespace Elastic.Apm.Helpers
+{
+	internal static class StringExtensions
+	{
+		internal static string TrimToLength(this string input, int maxLength)
```

Comment:
Hi @gregkalapos i thought about making it private, but i think it can be useful in other parts of code, for that library, for example if we`ll need to trim to custom length or smth like that, that's why i leaved it as internal

Reply:
👍 I don't have a strong opinion here... probably both options are fine. I see you made it private, let's keep it that way, unless you see it differently. We can make it internal once we need it later.